client/components/social-buttons/qr-code.tsx

```diff
@@ -17,17 +16,12 @@ type QrCodeLoginButtonProps = {
 	loginUrl: string;
 };
 
-export const QrCodeLoginButton = ( { loginUrl }: QrCodeLoginButtonProps ) => {
+export default function QrCodeLoginButton( { loginUrl }: QrCodeLoginButtonProps ) {
```

Comment:
Curiosity: why remove the named export and keep the default export? I would usually do the opposite, as I enjoy the devX (and tools-friendliness) of named exports.

Reply:
Yes, it probably should have been a named export. Historically, the Calypso convention was:
- export _unconnected_ (Redux) component as a named export. This is used by tests, which supply explicit values of the connected props, without having to mock Redux.
- export the connected (`connect( QrCodeLoginButton )`) component as default.

With Redux hooks, the unconnected component doesn't exist any more, and therefore only the default export makes sense.